week3/60-kvadratnoie-uravnieniie-2/task.py

```diff
@@ -0,0 +1,52 @@
+import math
+
+
+class rootCounts:
```

Comment:
is this Enum? then class RootCounts(Enum)?

class name must have UpperCase first letter

Reply:
done